### PR TITLE
Added Korean to locale

### PR DESCRIPTION
**Pull request by junukwon7**
Created ko_KR.yaml

### CODE REVIEW

File: packages/hydrooj/locales/ko_KR.yaml [hunk 1]
```diff
@@ -0,0 +1,832 @@
...
+Instead of copying the test data directly, the test data of the copied problems will be linked to the test data of the source problems (called the source test data). Thus, the copied problems can observe the changes in the source test data. The permissions of the test data of the copied problems follow the source test data, e.g., you still might not download them but the judges can. By uploading some new test data, the link will be broken and the new test data will be used.: Instead of copying the test data directly, the test data of the copied problems will be linked to the test data of the source problems (called the source test data). Thus, the copied problems can observe the changes in the source test data. The permissions of the test data of the copied problems follow the source test data, e.g., you still might not download them but the judges can. By uploading some new test data, the link will be broken and the new test data will be used.:
+Introduce must not exceed 500 characters and it will be shown in the list view.: 소개는 500자 미만이어야 합니다.
+Introduce: 소개
+Invalid password for user {0}.: 사용자 {0}에 대한 암호가 아닙니다.
+Invalid: 유효하지 않음
+Invitation Code: 초대 코드
+IO: IO
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: you can just remove these useless lines.

File: packages/hydrooj/locales/ko_KR.yaml [hunk 2]
```diff
@@ -0,0 +1,832 @@
...
+Edit own problem solution replies: 본인의 해설 댓글 수정
+Edit own problem solutions: 본인의 해설 수정
+Edit own problems: 본인의 문제 수정
+Edit own training plans: 본인의 학습 계획 수정
+Edit permissions of a role: 역할 권한 수정
+Edit problem solution replies: 해설 댓글 수정
+Edit problem solutions: 해설 수정
+Edit Problem: 문제 수정
+Edit problems: 문제 수정
+Edit Profile:  프로필 수정
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: extra space found

File: packages/hydrooj/locales/ko_KR.yaml [hunk 3]
```diff
@@ -0,0 +1,832 @@
...
+English: English
+Enroll Training: 학습 참여
+Enrolled: 참여함
+Enrollees: 참여 없음
+error: 에러
+Evaluated difficulty: 평가된 난이도
+expand: 확장
+Expire: 만료
+Export All Code: 모든 코드 내보내기
+Export as CSV: csv로 내보내기
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: maybe uppercase is better?

**Review by undefined-moe — CHANGES_REQUESTED** [review]

File: packages/hydrooj/locales/ko_KR.yaml [hunk 4]
```diff
@@ -0,0 +1,821 @@
...
+click to add: 추가
+CLOSE: 닫기
+Code blocks: 코드블럭
+Code copied to clipboard!: 복사 완료
+Code language: 언어
+Code: 코드
+collapse: 병합
+Comment: 댓글
+Comments: 댓글
+CommonMark Syntax: CommonMark Syntax
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: remove this line

File: packages/hydrooj/locales/ko_KR.yaml [hunk 5]
```diff
@@ -0,0 +1,821 @@
...
+'Hello, {0}! You can click following link to active your new email of your Hydro account:': 안녕하세요!，{0}！다음 링크를 눌러 이메일을 인증하세요：
+'Hello, {0}! You can click following link to reset the password of your Hydro account:': 안녕하세요，{0}！다음 링크를 눌러 비밀번호를 초기화하세요.：
+'Hello! You can click following link to sign up your Hydro account:': 다음 링크를 눌러 로그인하십시오. ：
+'Join {0}': '참여 {0}'
+'No': 아니오
+'Note: Problem title may not be hidden.': '주의: 문제 제목은 숨길 수 없습니다.'
+'Role {1} already exists in domain {0}.': '역할 {1}이(가) 이미 {0}에 존재합니다.'
+'Solved {0} problems, RP: {1} (No. {2})': '총 {0} 문제 해결.RP: {1} (No. {2})'
+'The value `{1}` of {0} already exists.': '{0} 의 값 `{1}` 이 이미 존재합니다.'
+'We use <a href="https://en.gravatar.com/" target="_blank">Gravatar</a> to present your avatar icon.': <a href="https://cn.gravatar.com/" target="_blank">Gravatar</a>를 만들어 보세요!'
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: why "!"

File: packages/hydrooj/locales/ko_KR.yaml [hunk 6]
```diff
@@ -0,0 +1,821 @@
...
+current domain: 그룹
+Current Email: 이메일
+Current Location: 위치
+Current password doesn't match.: 비밀번호가 틀렸습니다.
+Current Password: 현재 비밀번호
+Current Status: 현재 상태
+currently offline: 오프라인
+Data of Problem {0} not found.: 문제 {0}의 데이터가 없습니다.
+Data of problem {1} not found.: 문제 {1}의 데이터가 없습니다.
+Data of record {0} not found.: 평가 {0}의 데이터가 없습니다.
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml:186]: maybe a space after "{0}"? I'm not sure about Korean, but I thinks a long hexstring ObjectID without a space looks weird.

File: packages/hydrooj/locales/ko_KR.yaml [hunk 7]
```diff
@@ -0,0 +1,821 @@
...
+Destination Domain: 목표 그룹
+Development: 개발
+Difficulty Algorithm: 알고리즘 난이도
+Difficulty by admin: 관리자 난이도
+Difficulty by algorithm: 알고리즘 난이도
+Difficulty displayed: 난이도 표시
+Difficulty: 난이도
+Disabled: 비활성
+Disconnected from the server. Reconnecting...: 서버 연결이 끊어졌습니다. 재시도 중입니다...
+Discussion {1} not found.: 토론 {1}이(가) 없습니다.
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml:215]: ditto (also lines after)

File: packages/hydrooj/locales/ko_KR.yaml [hunk 8]
```diff
@@ -0,0 +1,821 @@
...
+Export as CSV: CSV로 내보내기
+Export as HTML: HTML로 내보내기
+Extension (days): 추가 (날짜)
+Extension Score Penalty: 추가 패널티
+Failed to join the domain. You are already a member.: 그룹 참여 실패. 이미 참여함.
+Feedback: 피드백
+Field {0} or {1} validation failed.: 항목 {0} 또는 {1} 검증 실패
+Field {0} validation failed.: 항목 {0} 검증 실패
+Filter: 필터
+Footer: Footer
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: remove

**Review by undefined-moe — CHANGES_REQUESTED** [review]

File: packages/hydrooj/locales/ko_KR.yaml (hunk 6 above)
Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml:186]: In Korean, it's kind of a suffix. There should be no space. Will it occur problems? (I found some lines without space in zh_CN too.)

File: packages/hydrooj/locales/ko_KR.yaml (hunk 5 above)
Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: Just suggesting to use gravatars. Removed it.

File: packages/hydrooj/locales/ko_KR.yaml (hunk 7 above)
Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml:215]: Also. It's like a suffix

File: packages/hydrooj/locales/ko_KR.yaml [hunk 9]
```diff
@@ -465,7 +465,7 @@ Only A-Z, a-z, 0-9 and _ are accepted: A-Z, a-z, 0-9 와 _만 허용됨
 Oops, there are no results.: 결과 없음
 Oops!: Oops!
 Open in Problem Set: 문제에서 열기
-Open Scratchpad: 연습장
+Open Scratchpad: 제출창
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: maybe it's better to translate it to "Online IDE" ? or "Open Editor" ?

Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: I think "Open Editor" will be better.

File: packages/hydrooj/locales/ko_KR.yaml [hunk 10]
```diff
@@ -465,7 +465,7 @@ Only A-Z, a-z, 0-9 and _ are accepted: A-Z, a-z, 0-9 와 _만 허용됨
 Oops, there are no results.: 결과 없음
 Oops!: Oops!
 Open in Problem Set: 문제에서 열기
-Open Scratchpad: 제출창
+Open Scratchpad: Editor 열기
 Open Since: '시작됨:'
```
Comment (undefined-moe) [inline packages/hydrooj/locales/ko_KR.yaml]: extra ":"

Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: Its because it is Korean.
Eng: Open Since 2020.11.17.
Kor: "2020.11.27.에 시작됨" or "시작: 2020.11.27"

Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: Will ```Open since {0}: '{0}에 시작됨'``` work?

Comment (undefined-moe) [reply packages/hydrooj/locales/ko_KR.yaml]: image

its a table(

Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: Oh.....  ok

Comment (junukwon7) [reply packages/hydrooj/locales/ko_KR.yaml]: oh.....

**Review by undefined-moe — APPROVED** [review]

**Comment by undefined-moe** [comment]
@all-contributors please add @junukwon7 for translation

**Pull request merged**